radar/2021-01-01/graalnaative.md

```diff
@@ -0,0 +1,14 @@
+---
+title:      "Graal Native Image"
+ring:       trial
+quadrant:   tools
+
+---
+Graal Native Image
```

Comment:
The name is not necessary, because it will be generated with the title from line 2